VOC2007/ImageSets/Main/aeroplane_train.txt

```diff
@@ -0,0 +1,2501 @@
+000012 -1
```

Comment:
You cannot copy any code/data to your repo.
On Mar 15, 2015 4:08 PM, "Mohamed Farghal" <mohamed@farghal.com> wrote:

> No. External files should be added to the repo. I will make a way to
> download them automatically. But a fundamental rule: External
> libraries/files/data must not be in the repo.
> On Mar 15, 2015 4:04 PM, "Hazem Amin" <notifications@github.com> wrote:
>
>> In VOC2007/ImageSets/Main/aeroplane_train.txt
>> <https://github.com/medo/image-categoriy-classifier/pull/11#discussion_r26447993>
>> :
>>
>> > @@ -0,0 +1,2501 @@
>> > +000012 -1
>>
>> @medo <https://github.com/medo> These files are the ones provided by
>> pascal and are needed to know what file belongs to what category. These
>> files have to be in the repo in order to train or test. They need to be
>> merged. Without these files we'll have the 10K images without knowing what
>> image belongs to what category. If you want us to have these files locally
>> without having them on the repo, how is any of you going to test it in
>> order to merge?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/medo/image-categoriy-classifier/pull/11/files#r26447993>
>> .
>>
>